### ISSUE TITLE
Reason for downsampling

### ISSUE THREAD

**Issue by jjparkcv** [Kai-46/ARF-svox2#6]
Hi Kai, I hope you're doing well -- this is JJ! Congratulations on this great work!

I have a question regarding some parts of the code.
[https://github.com/Kai-46/ARF-svox2/blob/22baf6e000e519b21ca26308f8ead1f4d2675cf5/opt/opt_style.py#L618](url)
In this line, I see that you are downsampling the rendered image to half to compute the losses.
But you could instead render an image with a lower resolution, which would be much more memory efficient.
Is there a reason for doing this downsampling? Did you want to do anti-aliasing, perhaps? 

Thanks in advance!

**Comment by Kai-46** [comment]
Good point! The downsampling step might be a bit ad-hoc there. If I remembered it correctly, I added the statement there mainly because when I was using the colorful triangle style, I'd like the triangles to be bigger in the stylized 3D appearance. In other words, the relative resolution between the content image and style image fed into the VGG feature extractor for calculating the NNFM loss can affect the style pattern size in the stylized appearance.
